torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -870,6 +872,10 @@ ProcessGroupNCCL::ProcessGroupNCCL(
       parseEnvVarIntDefault(NCCL_ASYNC_ERROR_HANDLING, 3 /*SkipCleanUp*/));
   desyncDebug_ = parseEnvVarFlag(NCCL_DESYNC_DEBUG) ||
       (dist_debug_level_ >= DebugLevel::Detail);
+  heartbeat_ = 1ULL;
+  monitorThreadEnabled_.store(parseEnvVarFlag(TORCH_NCCL_ENABLE_MONITORING));
+  heartbeatTimeoutInSec_ =
+      parseEnvVarIntDefault(TORCH_NCCL_HEARTBEAT_TIMEOUT, 60 * 2 /*2 Mins*/);
```

Comment:
nit: good practice to append _S after a variable name that represents time in seconds to avoid confusion.

Reply:
oh, sure, thanks for the suggestion.